management_api_app/api/routes/workspaces.py

```diff
@@ -3,23 +3,23 @@
 
 from api.dependencies.workspaces import get_repository, get_workspace_by_workspace_id_from_path
 from db.repositories.workspaces import WorkspaceRepository
-from models.domain.resource import Resource
-from models.schemas.resource import ResourcesInList, ResourceInResponse, ResourceInCreate
+from models.domain.workspace import Workspace
+from models.schemas.workspace import WorkspaceInCreate, WorkspaceIdInResponse, WorkspacesInList, WorkspaceInResponse
```

Comment:
I am a bit confused with the meaning of schema here. IS this request-response schemas and is it a fast api convention to call these schemas?

Reply:
yes and yes